contract_api/registry.py

```diff
@@ -362,6 +383,10 @@ def get_service_data_by_org_id_and_service_id(self, org_id, service_id):
             tags = self.repo.execute("SELECT tag_name FROM service_tags WHERE org_id = %s AND service_id = %s",
                                      [org_id, service_id])
 
+            media = self.get_service_media(org_id=org_id,service_id=service_id)
+            if media == None:
```

Comment:
Do this inside the get_service_media method. That way you dont have to check for None everytime you call this method

Reply:
this is addressed